src/nvidia_resiliency_ext/inprocess/abort.py

```
@@ -97,8 +100,27 @@ def shutdown_process_group_backend(group, device):
                 else:
                     backend.abort()
 
+    @staticmethod
+    def collect_fr_trace():
+        def _check_fr_env():
+            check_env_variables = ['TORCH_NCCL_DUMP_ON_TIMEOUT', 'TORCH_NCCL_ENABLE_MONITORING']
+            for env_var in check_env_variables:
+                if os.environ.get(env_var, '0') == '0':
+                    return False
```

Rather than silent failure, a one-time, single-rank info log will be informative for end users.